### DISCUSSION TITLE
Any release notes available?

### DISCUSSION THREAD

**Discussion by mstaal** [abersheeran/a2wsgi#55] [Q&A]
I just this day discovered a bug when using a2wsgi v1.8.0 along with FastAPI v0.110.0 (after having upgraded from v0.104.1). After some investigation, it turned out that the issue was resolved by bumpung a2wsgi to the most recent release, but I am curious what the issue was. Do you maintain any kind of release notes that could explain why there used to be a bug, and how it was resolved?

**Comment by abersheeran** [comment]
You can view git commit message 
![image](https://github.com/abersheeran/a2wsgi/assets/25364828/a512c938-f989-435a-b1a3-2d1a80b147f9)
